heartbeat/VirtualDomain

```diff
@@ -880,6 +930,14 @@ VirtualDomain_migrate_to() {
 			save_config
 		fi
 
+        if ocf_is_true "{OCF_RESKEY_start_resources}"; then
```

Comment:
Indentation for this section doesnt match the rest of the agent.

Reply:
fixed too